### PR TITLE
OpenShift CI - e2e tests

### PR DESCRIPTION
**Pull request by mgencur**
This PR is dependent on https://github.com/openshift/release/pull/2086
When running the tests from localhost 1 Go e2e test failed and the yaml e2e tests didn't finish because build-controller was throwing an error with invalid memory reference. But it doesn't happen immediately, it happens after some time. These problems need to be investigated.

### CODE REVIEW

File: openshift/e2e-tests-openshift.sh [hunk 1]
```diff
@@ -0,0 +1,191 @@
...
+    sed -e 's%\(.* image: \)\(github.com\)\(.*\/\)\(.*\)%\1 '"$registry_prefix"'\:build-\4%' | \
+    # process these images separately as they're passed as arguments to other containers
+    sed -e 's%github.com/knative/build/cmd/creds-init%'"$registry_prefix"'\:build-creds-init%g' | \
+    sed -e 's%github.com/knative/build/cmd/git-init%'"$registry_prefix"'\:build-git-init%g' | \
+    sed -e 's%github.com/knative/build/cmd/nop%'"$registry_prefix"'\:build-nop%g' \
+    >> $resolved_file_name
+  done
+}
+
+function enable_docker_schema2(){
```
Comment (alanfx) [inline openshift/e2e-tests-openshift.sh]: It seems like some of these functions should get extracted to a common location, but I'm not sure how to do that across the repos we have. I'll try to think of some way to do this, but open to ideas!

Comment (alanfx) [reply openshift/e2e-tests-openshift.sh]: Hey @mgencur, that sounds like a good plan to me! I am just glad there is a single place that we can share some common functions.

**Comment by mgencur** [comment]
The ci-operator doesn't build images correctly for an unknown reason. I will try to close this PR and open a new one.

**Pull request closed without merging**